posts/2021-04-19-JAXRS-multipart-payloads-21005-beta.adoc

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title: "Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta - OpenLiberty.io
+seo-description: With Open Liberty 21.0.0.5-beta you can now build multipart payloads for you JAX-RS client and services and also we now offer Jakarta EE 9 support for `oauth-2.0` and `samlWeb-2.0`.  
+blog_description: With Open Liberty 21.0.0.5-beta you can now build multipart payloads for you JAX-RS client and services and also we now offer Jakarta EE 9 support for `oauth-2.0` and `samlWeb-2.0`.  
```

Comment:
```suggestion
seo-description: With Open Liberty 21.0.0.5-beta you can now build multipart payloads for your JAX-RS client and services and also we now offer Jakarta EE 9 support for `oauth-2.0` and `samlWeb-2.0`.  
blog_description: With Open Liberty 21.0.0.5-beta you can now build multipart payloads for your JAX-RS client and services and also we now offer Jakarta EE 9 support for `oauth-2.0` and `samlWeb-2.0`.  
```

`and also we now offer` should probably be changed to something else.

Reply:
Changed to ```With Open Liberty 21.0.0.5-beta you can now build multipart payloads for your JAX-RS client and services and also present in this release is Jakarta EE 9 support for `oauth-2.0` and `samlWeb-2.0`.  ```